src/test/java/de/retest/recheck/ignore/SearchFilterFilesTest.java

```diff
@@ -18,11 +18,18 @@
 
 class SearchFilterFilesTest {
 
+	@Test
+	void getAllFilterFiles_should_find_all_filters() {
+		final List<Path> allFilterFiles = SearchFilterFiles.getAllFilterFiles();
+		assertThat( allFilterFiles.stream().map( path -> path.getFileName().toString() ) )
+				.contains( "positioning.filter", "visibility.filter" );
```

Comment:
I think you should add a user filter file (?) and test this, because it looks like this is currently the same test case as `getAllFilterFiles_should_find_all_filters()`.